src/commands/auth.ts

```diff
@@ -1,32 +1,120 @@
-import { open } from "openurl";
-import * as keytar from "keytar";
-import readline from "readline";
+import dotenv from 'dotenv';
+import open from "open";
+import readline from 'readline/promises';
+import fs from 'fs/promises';
+import path from 'path';
+import os from 'os';
+import crypto from 'crypto';
+import keytar from 'keytar';
+
+dotenv.config();
 
 const SERVICE_NAME = "AnyFlowCLI";
 const ACCOUNT_NAME = "apiToken";
+const ENCRYPTION_KEY = process.env.ENCRYPTION_KEY;
+
+if (!ENCRYPTION_KEY) {
+  console.error("ENCRYPTION_KEY is not set in the environment variables, run `anyflow init` to set it up.");
+  process.exit(1);
+}
 
 export async function authenticate() {
   console.log("Opening your browser to authenticate...");
 
-  const url = "https://anyflow.pro/api-tokens";
-  open(url, (err) => {
-    if (err) {
-      console.error(`Error opening URL: ${err}`);
-    }
-  });
+  const url = "https://app-staging.anyflow.pro/dev";
+
+  try {
+    await open(url);
+    console.log("Browser opened successfully.");
+  } catch (err) {
+    console.error(`Error opening URL: ${err}`);
+    process.exit(1);
```

Comment:
If the `open()` command fails the user will always fall into catch handler and exit the process, making it impossible to log in.

I prefer doing something like other CLIs: Show the user what URL we're trying to open and give the option for them to open themselves if the open() command fails.